cluster/v1beta1/types_placement.go

```diff
@@ -137,7 +137,6 @@ type GroupStrategy struct {
 	// group SHOULD be less than ClustersPerDecisionGroup. Once the number of items exceeds the ClustersPerDecisionGroup,
 	// the decisionGroups will also be be divided into multiple decisionGroups with same GroupName but different GroupIndex.
 	//
-	// +kubebuilder:validation:Required
```

Comment:
Why removing it ? we need to keep the default value 100% exist if the user did not defined the GroupStrategy. With omitempty available it will be empty string.

Reply:
okay, then sounds good to me.